dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -2084,9 +2086,9 @@
             <Setter.Value>
                 <ControlTemplate TargetType="InkToolbarPenConfigurationControl">
                     <Grid x:Name="RootElement" MinWidth="320">
-                        <Grid.ChildrenTransitions>
+                        <contract7Present:Grid.ChildrenTransitions>
```

Comment:
Grid.ChildrenTransitions should be available in older versions than this. This is fine to unblock the crash for now though.